product-scenarios/docs/how-to-run-scenario-tests-locally.md

```diff
@@ -0,0 +1,57 @@
+## How to Run Scenario Tests Locally
+
+`product-scenarios` directory holds the main scenarios that can be expected of an integration system and how each of those integration patterns can be achieved using WSO2 Enterprise Integrator. Each scenario contains the description of the scenario, along with the approaches that can be followed inorder to accomplish the scenario using WSO2 EI and test cases to verify the scenario.
+
+### Data Bucket
+
+The important point to remember when running scenario tests is that there should be a WSO2EI instance deployed and running prior to running the test cases. This brings in the capability of running EI instances locally as well as remotely, hence making it essential to define the service/backend URLs in advance. The directory which holds such configurations that define the deployment of the EI instance is known as the `DATA BUCKET` and it is necessary for it to contain the file, `deployment.properties` which contains all required URLs. You can have a look at [deployment.properties](../deployment.properties) to see a sample configuration for a setup that runs locally.
+
+### Test.sh
+
+The shell script file [test.sh](../test.sh) is responsible for triggering scenario tests. It triggers a maven build for all the scenarios upon execution of the file.
+ 
+### Steps to Run Tests against WSO2 Enterprise Integrator
+
+1. Copy lib jars located in [scenario-commons](../scenarios-commons/src/main/resources/lib) into `<EI_HOME>/lib` directory.
```

Comment:
We have to copy groovy-all-2.4.4.jar to ```<EI_HOME>/dropins``` directory.  Otherwise some tests in sub-scenario 1.6 will fail.

Reply:
groovy-all jar is inside ../scenarios-commons/src/main/resources/lib and copying it into <SERVER_HOME>/lib instead of dropins works as well. So shall we leave it as it is to make the users' work easier?